x-pack/docs/en/watcher/example-watches/example-watch-meetupdata.asciidoc

```diff
@@ -135,19 +144,28 @@ To set up the watch:
     }
   },
 --------------------------------------------------
+// NOTCONSOLE
 <1> Elasticsearch Date math is used to select the Logstash indices that contain the meetup data. The second pattern is needed in case the previous hour crosses days.
 <2> Find all of the RSVPs with `Open Source` as a topic.
 <3> Group the RSVPs by city.
 <4> Consolidate multiple RSVPs for the same event.
 <5> Sort the events so the latest events are listed first.
 <6> Group the events by name.
+--
```

Comment:
I assume that belongs to the two lines below?

Reply:
Ditto